linaro/linaro-4.9.4.cmake

```diff
@@ -0,0 +1,6 @@
+set(CMAKE_SYSTEM_NAME Linux)
+
+set(CMAKE_C_COMPILER   "${CMAKE_CURRENT_SOURCE_DIR}/linaro-4.9.4/bin/arm-linux-gnueabihf-gcc")
+set(CMAKE_CXX_COMPILER "${CMAKE_CURRENT_SOURCE_DIR}/linaro-4.9.4/bin/arm-linux-gnueabihf-g++")
```

Comment:
I would change the references to `CMAKE_CURRENT_SOURCE_DIR` to an environment variable defined by the docker image. The CMake syntax for referencing an environment variable is `$ENV{FOOBAR}`, where `FOOBAR` is of course the name of the environment variable.

Reply:
Similar to dockerfile qualms, I'd like to introduce as few new globals as possible, avoid setting a new standard. But I agree a shorter variable would clean this file up a lot. I could use a function to get a scoped variable, is that bad practice? Or would that scope the CMAKE_* variables too?

Reply:
> Similar to dockerfile qualms

Not sure what you're referring to here. New globals? Seems like environment variables describing the toolchain path would be a _good_ thing to have available globally, such that all users of this docker image (not only CMake users) have easy access to the compiler.